src/pages/Admin/Clusters/Detail/index.jsx

```diff
@@ -39,6 +39,8 @@ export default class ClusterDetail extends Component {
       user,
       match
     } = this.props;
+    clusterDetailStore.isHelmLoading = true;
```

Comment:
I think no need to add `isHelmLoading`, one store file just keep one loading state which means `is requesting data`